### PR TITLE
feature/#1567 rename Core serice to Orchestrator service

### PR DESCRIPTION
**Pull request by toan-quach**
#1567

### CODE REVIEW

File: taipy/core/__init__.py [hunk 1]
```diff
@@ -30,8 +30,9 @@
 
 !!! Note
 
-    Taipy Core provides a runnable service, `Core^` that runs as a service in a dedicated thread. The purpose is to
-    have a dedicated thread responsible for dispatching the submitted jobs to an available executor for their execution.
+    Taipy Orchestrator provides a runnable service, `Core^` that runs as a service in a dedicated thread.
```
Comment (jrobinAV) [inline taipy/core/__init__.py]:
```suggestion
    Taipy provides a runnable service, `Orchestrator^` that runs as a service in a dedicated thread.
```

File: taipy/core/__init__.py [hunk 2]
```diff
@@ -30,8 +30,9 @@
...
+    The purpose is to have a dedicated thread responsible for dispatching the submitted jobs to an available
+    executor for their execution.
 
     In particular, this `Core^` service is automatically run when Core is used with Taipy REST or Taipy GUI. See the
```
Comment (jrobinAV) [inline taipy/core/__init__.py]:
```suggestion
    In particular, this `Orchestrator^` service is automatically run when Core is used with Taipy REST or Taipy GUI. See the
```

File: taipy/core/__init__.py [hunk 3]
```diff
@@ -30,11 +30,12 @@
...
 
-    Taipy Core provides a runnable service, `Core^` that runs as a service in a dedicated thread. The purpose is to
-    have a dedicated thread responsible for dispatching the submitted jobs to an available executor for their execution.
+    Taipy provides a runnable service, `Orchestrator^` that runs as a service in a dedicated thread.
+    The purpose is to have a dedicated thread responsible for dispatching the submitted jobs to an available
+    executor for their execution.
 
-    In particular, this `Core^` service is automatically run when Core is used with Taipy REST or Taipy GUI. See the
-    [running services](../../../run-deploy/run/running_services.md) page of the user manual for more details.
+    In particular, this `Orchestrator^` service is automatically run when Core is used with Taipy REST or Taipy GUI.
```
Comment (jrobinAV) [inline taipy/core/__init__.py]:
```suggestion
    In particular, this `Orchestrator^` service is automatically run when used with Taipy REST or Taipy GUI.
```

**Review by jrobinAV — DISMISSED** [review]
last glitch.

**Review by jrobinAV — DISMISSED** [review]

**Review by trgiangdo — APPROVED** [review]
LGTM

**Comment by jrobinAV** [comment]
@toan-quach The doc build is failing because of some references to the Core class (`Core^`). Can you fix that in the same ticket or create another one please?

**Pull request merged**